functests/4_latency/latency.go

```diff
@@ -40,6 +40,7 @@ const (
 	defaultTestRuntime  = "300"
 	defaultMaxLatency   = -1
 	defaultTestCpus     = -1
+	MAX_INT             = 2147483647
```

Comment:
Capital letters are usually for environment variables.
please write it as ```maxInt```

Reply:
fixed , thanks !